tests/test_models.py

```diff
@@ -19,12 +19,15 @@ def tearDown(self):
         db.drop_all()
         self.app_context.pop()
 
-    def test_it_has_a_name(self):
-        plant = Plant(name='Gerald', plant_type='Celery')
-        self.assertTrue(plant.name == 'Gerald')
-        self.assertFalse(plant.name == 'Fred')
+    def test_it_has_a_plant_type(self):
+        plant = Plant(plant_type='Celery')
         self.assertTrue(plant.plant_type == 'Celery')
         self.assertTrue(plant.plant_type != 'Corn')
 
+    def test_it_has_a_plant_type(self):
+        plant = Plant(plant_type='Celery')
+        garden = Garden(plant_id=plant.id)
+        self.assertTrue(garden.plant_id == plant.id)
+
```

Comment:
Again, a chance to improve testing.  Testing seems to be a place where we're leaving some technical debt, but we're also testing in postman.  But as far as keeping it in mind for future iterations... or something we can leave to our interns!  haha